Makefile

```diff
@@ -132,28 +132,36 @@ dev-postgres13: start-postgres13
 
 start-pg-js: start-postgres13
 
-dev-pg-js: start-pg-js build-qe-napi build-connector-kit-js
+dev-pg-js: start-pg-js build-qe-napi build-driver-adapters-kit
 	cp $(CONFIG_PATH)/pg-js $(CONFIG_FILE)
 
 test-pg-js: dev-pg-js test-qe-st
 
-dev-pg-wasm: start-pg-js build-qe-wasm build-connector-kit-js
+dev-pg-wasm: start-pg-js build-qe-wasm build-driver-adapters-kit
 	cp $(CONFIG_PATH)/pg-wasm $(CONFIG_FILE)
 
 test-pg-wasm: dev-pg-wasm test-qe-st
 
 test-driver-adapter-pg: test-pg-js
 test-driver-adapter-pg-wasm: test-pg-wasm
 
+start-pg-bench:
+	docker compose -f query-engine/driver-adapters/executor/bench/docker-compose.yml up --wait -d --remove-orphans postgres
```

Comment:
Builds a custom image that loads the seed file, see Dockerfile at https://github.com/prisma/prisma-engines/pull/4567/files#diff-96c0901b002d607e8553d7b5b38b2dd0dccdf904ade81ff4c8f4dffa6ab54140